governance-exercises/Interpretation Exercise Response.md

```diff
@@ -0,0 +1,46 @@
+**Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?**
+
+Yes, recent events over the EDSR made it very clear ambiguity exists. Various community members were unsure on if this clause would allow for such a vote. One reasonable interpretation is that this power was meant to be used to fix obvious errors or weird edge cases that break things, but the author clarified that it could be used to deal with broader protocol issues(such as low DSR utilization and declining DAI supply). 
+
+Given the author's intent, I would suggest  the following edits be included in a future Scopes update to clarify that the power is intended to be this broad. To be clear, this does not necessarily mean I believe Governance Facilitators ought to have this power, but that this is the intent of of the author and majority of voting MKR.
```

Comment:
Okay, I kind of get that, but as Governance Facilitator, you're the one that needs to push back if you think the governance facilitator role shouldn't have a specific power. Even if large token holders disagree with you.